src/components/scripts/DirectionButtons.js

```diff
@@ -0,0 +1,63 @@
+
+
+
+//--Buttons that helps player to move in different directions--//
+
+function DirectionButtons({
+  moveNorth,
+  moveSouth,
+  moveEast,
+  moveWest,
+  returnToStart,
+  buttonState,
+}) {
+  return (
+    <>
+
+      {/* as these buttons are clicked the marker moves N, S, E, W accordingly */}
+      {/* North Button */}
+      <button
+        className="buttonStyle"
+        onClick={moveNorth}
+        disabled={!buttonState}
```

Comment:
This is a good way to track button state for the 3 main game buttons. Since you know that two buttons will always be the opposite of the start button, creating one overall `disabled` state to check against, saves you from tracking 3 individual button states.